test/integration_test.dart

```diff
@@ -128,6 +128,7 @@ void coreTests() {
 
         final options = _getOptionsFromString(src);
         var configuredLints =
+            // ignore: cast_nullable_to_non_nullable
             (options['linter'] as YamlMap)['rules'] as YamlList;
```

Comment:
This seems like a false positive to me. IIRC, `options` is a `Map<String, dynamic>`. While it's true that `dynamic` is nullable, I think it's reasonable to cast it to either a nullable or non-nullable type.

Reply:
fyi @a14n